### PR TITLE
parametric/otel tests: fixed wrong asssertion, must set span kind not type

### PR DESCRIPTION
**Pull request by dianashevchenko**
## Description
Relates to / Noticed in https://github.com/DataDog/dd-trace-go/pull/2334

## Motivation
Previously, we were wrongfully setting span.type based on the span.kind attribute. Now, span.kind will map to span.kind.

## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

Once your PR is reviewed, you can merge it! :heart:

## Reviewer checklist

* [ ] Check what scenarios are modified. If needed, add the relevant label (`run-parametric-scenario`, `run-profiling-scenario`...). If this PR modifies any system-tests internal, then add the `run-all-scenarios` label ([more info](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md)). 
* [ ] CI is green
   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] if any of `build-some-image` label is present
  1. is the image labl have been updated ? 
  2. just before merging, locally build and push the image to hub.docker.com
* [ ] if a scenario is added (or removed), add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

**Review by darccio — APPROVED** [review]
LGTM 😁

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
There are some failures related with this change

**Comment by dianashevchenko** [comment]
@robertomonteromiguel since I'm fixing the test itself, it is expected to have some failures. It should pass on Go however, as it is merged already

File: tests/apm_tracing_e2e/test_otel.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def test_datadog_otel_span(self):
         assert child.get("parentID") == parent.get("spanID")
         assert child.get("spanID") != "10000"
         assert child.get("duration") == "1000000000"
-        assert child.get("type") == "internal"
+        assert child.get("meta").get("span.kind") == "internal"
```
Comment (robertomonteromiguel) [inline tests/apm_tracing_e2e/test_otel.py:41]: You should annotate this method with @missing_feature for current golang prod release and for Java (see CI failures).

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]

**Review by robertomonteromiguel — APPROVED** [review]

**Pull request merged**